static/robots.txt

```diff
@@ -1,2 +1,4 @@
 User-agent: *
-Disallow: /
\ No newline at end of file
+Disallow: /.build-info
```

Comment:
Based on [this Google Devs doc](https://developers.google.com/search/docs/crawling-indexing/robots/robots_txt#url-matching-based-on-path-values), I think we could also do this:
```suggestion
Disallow: /.
```
if we wanted to ignore any path under the root that begins with `.`.  The period doesn't need to be escaped since it doesn't have a special meaning to the parsers like it does with regular expressions, and we don't need a wildcard like `/.*` since trailing wildcards are ignored (see `/fish*` example in the docs).